### PR TITLE
fix: consistently remove trailing slashes

### PR DESCRIPTION
**Pull request by pjft**
Addressing #17 , but unsure if it's just on my setup or if it'll break other things for others. Just thought I'd share here.

### CODE REVIEW

File: nextcloud-inotifyscan [hunk 1]
```diff
@@ -188,8 +188,8 @@ def watch_instances(instances):
               user = None
               _logger.warning('cannot find valid user for %s', data_prefix)
         if user is not None:
-          path_prefix = '/'+user+'/files'+storage['mount_point']
-          data_prefix = storage['configuration']['datadir']
```
Comment (Blaok) [inline nextcloud-inotifyscan:192]: #17 was caused by potential inconsistency here: `storage['mount_point']` ends with `/` but `storage['configuration']['datadir']` does not. Consistently removing trailing `/` should solve the problem.

File: nextcloud-inotifyscan [hunk 2]
```diff
@@ -150,7 +150,7 @@ def watch_instances(instances):
       # path_prefix is the virtual folder recognized by Nextcloud
       # data_prefix is the actual filesystem folder
       # both should point to the same logical folder
-      path_prefix = '/'+user+'/files'
+      path_prefix = '/'+user+'/files/'
```
Comment (Blaok) [inline nextcloud-inotifyscan]: This works but is not necessary. I reverted this change to make all prefixes consistently end without trailing `/`.

**Comment by Blaok** [comment]
Thanks for the PR! This fixes #17.

File: nextcloud-inotifyscan (hunk 2 above)
Comment (pjft) [reply nextcloud-inotifyscan]: Thanks! You know the logic better than I do - glad it was helpful:)

**Pull request merged**